modules/http-api/src/main/java/org/mule/runtime/http/api/sse/SseSource.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.sse;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoImplement;
+
+import java.util.function.Consumer;
+
+/**
+ * A consumer of server-sent events.
+ * <p>
+ * This API is EXPERIMENTAL. Do not use it until it is stable.
+ * 
+ * @since 4.9.3, 4.10.0
+ */
+@Experimental
+@NoImplement
+public interface SseSource {
+
+  int READY_STATUS_CONNECTING = 0;
+  int READY_STATUS_OPEN = 1;
+  int READY_STATUS_CLOSED = 2;
```

Comment:
Use an enum instead

Reply:
The spec defines numeric values actually